src/main/kotlin/dev/stalla/model/podcastindex/GeographicLocation.kt

```diff
@@ -132,7 +132,21 @@ public class GeographicLocation public constructor(
     }
 
     /** Checks if `this` type matches a [pattern] type taking parameters into account. */
-    public fun match(pattern: String): Boolean = match(of(pattern))
+    public fun matches(pattern: String): Boolean = matches(of(pattern))
+
+    /** Checks if `this` type matches a [pattern] type taking parameters into account. */
+    @Deprecated(
+        message = "This method is scheduled for removal in v2.0.0",
+        replaceWith = ReplaceWith("matches")
+    )
+    public fun match(pattern: GeographicLocation?): Boolean = matches(pattern)
```

Comment:
Given we haven't released this class yet, should we just do the rename?

Reply:
Haha, yep that makes more sense 😄